### PR TITLE
SCSS - Edit Basic Info of Profile

### PR DESCRIPTION
**Pull request by alenaminsk**

### CODE REVIEW

File: src/test/java/ServiceNow/SCSS/Steps/ApplicantFlowSteps.java [hunk 1]
```diff
@@ -1,8 +1,121 @@
 package ServiceNow.SCSS.Steps;
 
 import appsCommon.PageInitializer;
+import com.nci.automation.web.CommonUtils;
 import io.cucumber.java.en.Given;
+import org.junit.Assert;
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/SCSS/Steps/ApplicantFlowSteps.java:6]: You could use the existing Assert in CommonUtils class and avoid additional imports. Also, seems like you are using Junit and Testng assertion interchangeably.

File: src/test/java/CustomBusiness/ETD/Features/ETDEmployee.feature [hunk 2]
```diff
@@ -6,15 +6,15 @@ Given User is on ETD home page and user is "AALAI, MARRIAM"
 When User clicks on About ETD Tab
 Then employee will see a respective message
 
-@validateTierChangesNonEmployee @Regression @Needs_review
+@validateTierChangesNonEmployee @Needs_review @Regression
```
[the remark this answers is not in the document]
Comment (alenaminsk) [reply src/test/java/CustomBusiness/ETD/Features/ETDEmployee.feature:9]: These @Needs_review were put by another QA as these scenarios are not up to date.

File: src/test/java/CustomBusiness/ETD/StepsImplementation/ETDFlowStepsImpl.java [hunk 3]
```diff
@@ -1,30 +1,27 @@
...
 import com.nci.automation.web.EnvUtils;
 import com.nci.automation.web.JavascriptUtils;
 import com.nci.automation.web.WebDriverUtils;
 import com.nci.automation.xceptions.TestingException;
-import appsCommon.PageInitializer;
+import java.util.List;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
```
[the remark this answers is not in the document]
Comment (alenaminsk) [reply src/test/java/CustomBusiness/ETD/StepsImplementation/ETDFlowStepsImpl.java:15]: I ran regression on ETD. Not working on it now.

File: src/test/java/ServiceNow/SCSS/Steps/ApplicantFlowSteps.java (hunk 1 above)
Comment (alenaminsk) [reply src/test/java/ServiceNow/SCSS/Steps/ApplicantFlowSteps.java:6]: Applied Test NG assertion from CommonUtils

File: src/test/java/ServiceNow/SCSS/Steps/ApplicantFlowSteps.java [hunk 4]
```diff
@@ -1,8 +1,121 @@
...
+    @Given("User clicks Edit for {string} section")
+    public void user_clicks_edit_for_section(String section) {
+        applicantFlowStepsImplementation.clickEditButton(section);
+    }
+
+    @Given("User inputs {string} into first name field")
+    public void user_inputs_into_first_name_field(String firstName) {
+        CommonUtils.waitForClickability(applicantFlowPage.fldFirstNameProfile);
+        CommonUtils.sendKeys(applicantFlowPage.fldFirstNameProfile, Keys.chord(Keys.CONTROL,"a", Keys.DELETE));
+        CommonUtils.sendKeys(applicantFlowPage.fldFirstNameProfile,firstName);
```
[the remark this answers is not in the document]
Comment (alenaminsk) [reply src/test/java/ServiceNow/SCSS/Steps/ApplicantFlowSteps.java:26]: Implemented in Steps Implementation class

**Review by hamidnesar — APPROVED** [review]

**Review by georgeb79 — APPROVED** [review]

**Review by sarwarahmed1 — APPROVED** [review]

**Review by iamdez99 — APPROVED** [review]
Reviewed and approved - juarezds

**Pull request merged**